lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2653,7 +2650,7 @@ static bool ShouldEnableCXXInterop(CompileUnit *cu) {
       if (unit.get() == cu) {
         if (cu->GetLanguage() == eLanguageTypeSwift)
           for (const char *arg : args.GetArgumentArrayRef())
-            if (strcmp(arg, "-enable-experimental-cxx-interop") == 0)
+            if (strcmp(arg, flag) == 0)
```

Comment:
Do we not have a DWARF attribute for this?

Reply:
We don't